packages/test/test-version-utils/package.json

```diff
@@ -34,21 +44,21 @@
 		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
-		"test:mocha": "mocha  --recursive \"dist/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup --timeout=10000",
-		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
-		"tsc": "tsc"
+		"test:mocha": "npm run test:mocha:esm",
+		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
```

Comment:
@jason-ha - Keep the '--timeout=10000' from the original 'test:mocha'?

Reply:
That would make sense to me. Another reason not to split when just one flavor - easier to see what changed. :)